src/posts/donate.md

```diff
@@ -1,39 +1,40 @@
 ---
 title: Donate to PauseAI
-description: With your financial support we can have a bigger impact.
+description: Become a donor today and fuel our global fight to pause unsafe AI!
 ---
 
-We started PauseAI as a group of volunteers, who work on it in their free time.
-Since February 2024, PauseAI is a [registered non-profit foundation](/legal).
-We do not yet have sufficient funds to hire people to work full-time on PauseAI, so we still run on volunteer work.
-With your financial support, we can professionalize and have a bigger impact.
+Become a donor today and fuel our worldwide fight against unsafe AI. Your donation powers our global awareness campaigns, protests, and lobbying to pause unsafe AI. Every donation, no matter the size, makes a difference in our urgent cause!
+
+Note: Ideally have two donate buttons, giving a choice between donating in Dollars or Euros -- it might help increase donations from Americans, and also, giving people a choice is shown to help increase donations 
```

Comment:
Whoops, this should have been a comment I presume.

Anyways, I'm not sure why having seperate buttons for currencies make sense. Is this a common thing? Haven't seen it yet anywhere else